File: app/src/main/java/com/example/asiantech/playmusic/adapter/PlaySongAdapter.java
package com.example.asiantech.playmusic.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import com.example.asiantech.playmusic.fragment.LyricsFragment_;
import com.example.asiantech.playmusic.fragment.PlayListFragment_;
import com.example.asiantech.playmusic.fragment.VisualizerFragment_;

public class PlaySongAdapter extends FragmentPagerAdapter {
    public PlaySongAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        Fragment fragment = null;
        switch (position) {
            case 0:
                fragment = new PlayListFragment_();
                break;
            case 1:
                fragment = new VisualizerFragment_();
                break;
            case 2:
                fragment = new LyricsFragment_();
                break;
        }
        return fragment;
    }

    @Override
    public int getCount() {
        return 3;
    }
}

File: app/src/main/java/com/example/asiantech/playmusic/adapter/PlaySongAdapter.kt
package com.example.asiantech.playmusic.adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter
import com.example.asiantech.playmusic.fragment.LyricsFragment
import com.example.asiantech.playmusic.fragment.PlayListFragment
import com.example.asiantech.playmusic.fragment.VisualizerFragment

/**
 * PlaySongAdapter.
 *
 * @author HoaHT
 */
class PlaySongAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

    override fun getItem(position: Int): Fragment? {
        var fragment: Fragment? = null
        when (position) {
            0 -> fragment = PlayListFragment()
            1 -> fragment = VisualizerFragment()
            2 -> fragment = LyricsFragment()
        }
        return fragment
    }

    override fun getCount(): Int {
        return 3
    }
}